### DISCUSSION TITLE
App which prevents SAF usage

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#10821] [Show and tell]
Just for telling and in case we search for it later on:

I had a user contacting us with the problem the he is unable to select a base directory as the shown directory browser was totally empty. At first sight it looked like the known problem we mitigated meanwhile (show advanced options). But as it was an Android 8 device from Sony, which had not shown any problems (for other users) before, it was even more unlikely to be caused by this.

After sharing some screenshots I stumbled upon a settings item in that app saying "Buy No-Ad version", which pretty much means, that this is not the system app showing up here.
Indeed the user was able to identify it was an app with package name `com.scavengers.apps.filemanager` he installed several years ago which just catched this intent.

After deinstalling it, the base directory selection in c:geo as well as similar problems the user had in other apps were fixed.

**Comment by eddiemuc** [comment]
Interesting case, and good to know! Thanks for sharing.

I wonder why Android even allows other apps to catch these very special intents.
